tests/test_bai.py

```diff
@@ -68,6 +68,7 @@ def test_author_tokenizer(self):
         token_list = author_tokenize("Ellis Richard")
         self.assertEqual(token_list, ['E Richard',
                                       'Ellis Richard',
+                                      'Richard',
```

Comment:
That's why I added this line:
https://github.com/Panos512/inspire-next/blob/authors_lowercasing/inspirehep/modules/authors/utils.py#L206 .

This adds the last names to the name variations to support 'only last-name searches'.